docs/commerce/checkout/customize_checkout.md

```diff
@@ -0,0 +1,202 @@
+---
+description: Customize the existing checkout functionality to support additional functions.
+edition: commerce
+---
+
+# Customize checkout
+
+When you work with your Commerce implementation, you can review and modify 
+the checkout configuration.
+
+Checkout is an essential component of the Commerce offering.
+It collects data that is necessary to create an order, including:
+
+- payment method
+- shipping method
+- billing / delivery address
+
+It could also collect any other information that you find necessary.
+
+Depending on your needs, the checkout process can be either complicated or very simple. 
+For example, if the website is selling airline tickets, you may need several additional steps 
+with passengers defining their special needs.
+On the other side of the spectrum would be a store that sells books with personal pickup, 
+where one-page checkout would be enough.
+
+There are several factors that make checkout particularly flexible and customizable:
+
+- it is based on Symfony workflow
+- exposes a variety of APIs
+- exposes Twig functions that help you render the steps
+
+The most important contract exposed by the package is the `CheckoutServiceInterface` interface. 
+It exposes a number of methods that you can call, for example, to load checkouts based 
+on checkout identifier or for a specific cart. 
+Other methods help you create, update, or delete checkouts. 
+
+For more information, see [Checkout API](checkout_api.md).
+
+## Add checkout stage
+
+By default [[= product_name =]] comes with a multi-step checkout process, which you can expand by adding steps.
+For example, if you were creating a project for selling theater tickets, you could add a step 
+that allows users to select their seats.
+
+### Define workflow
+
+You can create workflow definitions in the `config/packages/ibexa.yaml` file. 
+Each workflow definition consists of a series of stages as well as a series of transitions between the stages. 
```

Comment:
I'm wondering if it's worth mentioning here that this workflow is Symfony workflow and different from the editorial workflow. Not sure.

Reply:
Definitely, Ibexa Workflow operates only in the context of content objects which isn't the case in the checkout.

Reply:
@DominikaK I am mentioning it in line 28, do you think it is necessary to repeat this infornation here?